src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.FSharp.xaml

```diff
@@ -0,0 +1,60 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Rule Name="Build"
+      OverrideMode="Extend"
+      xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <EnumProperty Name="Nullable" Category="General" Visible="False" />
+  <BoolProperty Name="AllowUnsafeBlocks" Category="General" Visible="False" />
+
+  <BoolProperty Name="Optimize"
+                DisplayName="Optimize code"
+                Description="Enable compiler optimizations for smaller, faster, and more efficient output."
+                Category="General">
+    <BoolProperty.Metadata>
+      <NameValuePair Name="SearchTerms" Value="optimise;optimisation" />
+    </BoolProperty.Metadata>
+  </BoolProperty>
+
+  <BoolProperty Name="Tailcalls"
+              DisplayName="Generate tail calls"
+              Description="Enable tail calls generation."
+              Category="General">
+    <BoolProperty.Metadata>
+      <NameValuePair Name="SearchTerms" Value="tailcall;tail;tail calls" />
+    </BoolProperty.Metadata>
+  </BoolProperty>
+
+  <StringProperty Name="OtherFlags"
+                DisplayName="Other flags"
+                Description="Additional compiler flags."
+                Category="General" />
+
+  <EnumProperty Name="DebugType"
+              DisplayName="Debug symbols"
+              Description="Specifies the kind of debug symbols produced during build."
+              Category="General">
+    <EnumProperty.DataSource>
+      <DataSource HasConfigurationCondition="False"
+                Persistence="ProjectFileWithInterception" />
+    </EnumProperty.DataSource>
+    <EnumProperty.Metadata>
+      <NameValuePair Name="SearchTerms" Value="debug type" />
+    </EnumProperty.Metadata>
+    <EnumValue Name="none" DisplayName="No symbols are emitted" />
+    <EnumValue Name="pdbonly" DisplayName="PDB Only" />
```

Comment:
For C#/VB projects `pdbonly` has been deprecated:

https://github.com/dotnet/project-system/blob/5a60b3dad0598cf98fbdfb6cdaccc68adc8b43e4/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml#L139-L143

Is this different for F# projects? If so, the interceptor used here will mess with that. If not, please mirror these.

Again, if this is intentionally different to that which it overrides, please explain why in a comment.

Reply:
Yeah, on coreclr, pdbonly will be the same as full, gonna fix it. Thanks.